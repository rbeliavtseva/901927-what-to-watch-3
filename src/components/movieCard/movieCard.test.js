import React from "react";
import renderer from "react-test-renderer";
import MovieCard from "./movieCard.jsx";

// Mocks
const movie = {
  title: `Inception`,
  genre: `Science Fiction`,
  year: 2010
};

it(`Render MovieCard`, () => {
  const tree = renderer
    .create(<MovieCard
      movie={movie}
      onMovieCardHover={()=>{}}
      onMovieCardTitleClick={()=>{}}
    />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
