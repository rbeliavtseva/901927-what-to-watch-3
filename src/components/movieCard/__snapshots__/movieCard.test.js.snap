// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render MovieCard 1`] = `
<article
  className="small-movie-card undefined"
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
>
  <a
    href="/dev-film"
    onClick={[Function]}
  >
    <div
      className="small-movie-card__image"
      onClick={[Function]}
    >
      <img
        alt="Inception"
        height="175"
        src="http://picsum.photos/id/1005/300/150"
        width="280"
      />
    </div>
  </a>
  <h3
    className="small-movie-card__title"
    onClick={[Function]}
  >
    <a
      className="small-movie-card__link"
      href="/dev-film"
      onClick={[Function]}
    >
      Inception
    </a>
  </h3>
</article>
`;
